### PR TITLE
zh: update for alert and monitoring

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)
Update doc for alert and monitoring

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/monitor-a-tidb-cluster.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ kubectl port-forward -n <namespace> svc/<release-name>-grafana 3000:3000 &>/tmp/
 
 然后在浏览器中打开 [http://localhost:3000](http://localhost:3000)，默认用户名和密码都为 `admin`。
 
-Grafana 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持负载均衡器，你可以在 `values.yaml` 中将 `monitor.grafana.service.type` 修改为 `LoadBalancer`，然后在执行 `helm upgrade` 后通过负载均衡器访问面板。
+也可以参考[通过 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)，设置 `spec.grafana.service.type` 为 `NodePort` 或者 `LoadBalancer`，通过 `NodePort` 或者 `LoadBalancer` 查看监控面板。
 
-如果不需要使用 Grafana，可以在部署时在 `values.yaml` 中将 `monitor.grafana.create` 设置为 `false` 来节省资源。这一情况下需要使用其他已有或新部署的数据可视化工具直接访问监控数据来完成可视化。
+如果不需要使用 Grafana，可以在部署时将 `TidbMonitor` 中 `spec.grafana` 部分删除。这一情况下需要使用其他已有或新部署的数据可视化工具直接访问监控数据来完成可视化。
```
Comment (ran-huang) [inline zh/monitor-a-tidb-cluster.md]:
```suggestion
如果不需要使用 Grafana，可以在部署时将 `TidbMonitor` 中的 `spec.grafana` 部分删除。这一情况下需要使用其他已有或新部署的数据可视化工具直接访问监控数据来完成可视化。
```

File: zh/monitor-a-tidb-cluster.md [hunk 2]
```diff
@@ -85,11 +85,14 @@ Prometheus 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持
 
 在随 TiDB 集群部署 Prometheus 时，会自动导入一些默认的报警规则，可以通过浏览器访问 Prometheus 的 Alerts 页面查看当前系统中的所有报警规则和状态。
 
-我们目前暂不支持报警规则的自定义配置，如果确实需要修改报警规则，可以手动下载 charts 进行修改。
+我们目前支持报警规则的自定义配置，可以参考下面步骤修改报警规则：
+
+- 参考[通过 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)，在为 TiDB 集群部署监控的过程中，设置 `spec.reloader.service.type` 为 `NodePort` 或者 `LoadBalancer`。
+- 通过 `NodePort` 或者 `LoadBalancer` 访问 reloader 服务，点击上方 `Files` 选择要修改的报警规则文件进行修改，修改完成后 `Save`。
```
Comment (ran-huang) [inline zh/monitor-a-tidb-cluster.md]: If these two steps should be taken in order, please use `1, 2,...` to mark the order. 
If the user only has to take either one of the steps, please also specify in the document.

**Comment by DanielZhangQD** [comment]
@ran-huang comments addressed, PTAL.

File: zh/monitor-a-tidb-cluster.md [hunk 3]
```diff
@@ -85,11 +85,14 @@ Prometheus 服务默认通过 `NodePort` 暴露，如果 Kubernetes 集群支持
...
-我们目前暂不支持报警规则的自定义配置，如果确实需要修改报警规则，可以手动下载 charts 进行修改。
+我们目前支持报警规则的自定义配置，可以参考下面步骤修改报警规则：
+
+1. 参考[通过 TidbMonitor 监控 TiDB 集群](monitor-using-tidbmonitor.md)，在为 TiDB 集群部署监控的过程中，设置 `spec.reloader.service.type` 为 `NodePort` 或者 `LoadBalancer`。
+2. 通过 `NodePort` 或者 `LoadBalancer` 访问 reloader 服务，点击上方 `Files` 选择要修改的报警规则文件进行修改，修改完成后 `Save`。
 
 默认的 Prometheus 和报警配置不能发送报警消息，如需发送报警消息，可以使用任意支持 Prometheus 报警的工具与其集成。推荐通过 [AlertManager](https://prometheus.io/docs/alerting/alertmanager/) 管理与发送报警消息。
 
-如果在你的现有基础设施中已经有可用的 AlertManager 服务，可以在 `values.yaml` 文件中修改 `monitor.prometheus.alertmanagerURL` 配置其地址供 Prometheus 使用；如果没有可用的 AlertManager 服务，或者希望部署一套独立的服务，可以参考官方的[说明](https://github.com/prometheus/alertmanager)部署。
+如果在你的现有基础设施中已经有可用的 AlertManager 服务，可以参考[设置 kube-prometheus 与 AlertManager](monitor-using-tidbmonitor.md#设置-kube-prometheus-与-AlertManager) 设置 `spec.alertmanagerURL` 配置其地址供 Prometheus 使用；如果没有可用的 AlertManager 服务，或者希望部署一套独立的服务，可以参考官方的[说明](https://github.com/prometheus/alertmanager)部署。
```
Comment (lilin90) [inline zh/monitor-a-tidb-cluster.md]: @DanielZhangQD [This link](https://pingcap.com/docs-cn/tidb-in-kubernetes/v1.1/monitor-using-tidbmonitor/#%E8%AE%BE%E7%BD%AE-kube-prometheus-%E4%B8%8E-AlertManager) cannot jumps to the specified position on the PingCAP website. The correct version is:

`可以参考[设置 kube-prometheus 与 AlertManager](monitor-using-tidbmonitor.md#设置-kube-prometheus-与-alertmanager) `

Would you please fix it by creating another PR?

**Pull request merged**